lib/cloudkeeper/aws/filter_helper.rb

```diff
@@ -0,0 +1,39 @@
+module Cloudkeeper
+  module Aws
+    # Class used for generating filter structures used by AWS .describe_tags
+    class FilterHelper
```

Comment:
Also these generic methods are great and you can use them but maybe write some more specific methods just for what you need since you're pretty much repeating 3 calls in `CoreConnector` anyway.